TLM/TLM/TLM.csproj

```diff
@@ -12,7 +12,7 @@
     <TargetFrameworkVersion>v3.5</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
     <TargetFrameworkProfile />
-    <LangVersion>latest</LangVersion>
+    <LangVersion>preview</LangVersion>
```

Comment:
why?

Reply:
because my compiler thinks that C#9.0 is not the `latest`.
I'm not merging it yet so I'll try to do something with that.

Reply:
And here I thought you wanted to stay on c#7.3 ^^